packages/eip-1193-provider/README.md

```diff
@@ -0,0 +1,120 @@
+# Turnkey EIP-1193 Provider
+
+The `@turnkey/eip-1193-provider` package delivers a Turnkey-compatible Ethereum provider that adheres to the EIP-1193 standards. It's built to integrate seamlessly with a broad spectrum of EVM-compatible chains, offering capabilities like account management, transaction signing, and blockchain interaction. This initial setup is intended for use in conjunction with the [`@turnkey/http`](https://www.npmjs.com/package/@turnkey/http) and [`@turnkey/webauthn-stamper`](https://www.npmjs.com/package/@turnkey/webauthn-stamper) package, for initial authentication.
+
+## Installation
+
+Before you start using the Turnkey EIP-1193 Provider, make sure to install the necessary packages in your project. This guide assumes you have a Node.js environment ready for development.
+
+Install the required packages using NPM or Yarn:
+
+```bash
+npm install @turnkey/eip-1193-provider @turnkey/http @turnkey/webauthn-stamper
+```
+
+```bash
+pnpm add @turnkey/eip-1193-provider @turnkey/http @turnkey/webauthn-stamper
+```
+
+```bash
+yarn add @turnkey/eip-1193-provider @turnkey/http @turnkey/webauthn-stamper
```

Comment:
do you have a preference between npm/pnpm/yarn? historically we've used pnpm everywhere

Reply:
This was just to give developers different options, but I personally prefer pnpm now!